File: chroma-league/src/test/java/com/bonepl/chromaleague/hud/animations/EnemyHeraldKillAnimationTest.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.hud.AnimationTester;
import org.junit.jupiter.api.Test;

class EnemyHeraldKillAnimationTest {

    @SuppressWarnings("JUnitTestMethodWithNoAssertions")
    @Test
    void playEnemyHeraldKillAnimation() {
        new AnimationTester().testAnimation(new EnemyHeraldKillAnimation());
    }
}

File: test/hud/animations/EnemyHeraldKillAnimation.test.ts
import {describe, test} from 'vitest';
import {AnimationTester} from '../../AnimationTester.js';
import {EnemyHeraldKillAnimation} from '../../../src/hud/animations/EnemyHeraldKillAnimation.js';

/**
 * Integration test for enemy herald kill animation on actual hardware.
 */
describe('EnemyHeraldKillAnimation Integration', () => {
    test('playEnemyHeraldKillAnimation', async () => {
        await new AnimationTester().testAnimation(new EnemyHeraldKillAnimation());
    }, 30000);
});
